### ISSUE TITLE
documentation for math pow

### ISSUE THREAD

**Issue by dvd101x** [josdejong/mathjs#2884]
Hi, I encountered a minor issue in the documentation for pow.

While evaluating `help(pow)` two of the examples show an error

```
math.pow([[1, 2], [4, 3]], 2)
    Error: Undefined symbol math
math.pow([[1, 2], [4, 3]], -1)
    Error: Undefined symbol math
```

A possible fix is to remove `math.`

```
pow([[1, 2], [4, 3]], 2)
    [[9, 8], [16, 17]]
pow([[1, 2], [4, 3]], -1)
    [[-0.6, 0.4], [0.8, -0.2]]
```
## Steps to replicate

open the main demo and evaluate `help(pow)`

**Comment by josdejong** [comment]
Thanks David 👍 . I've fixed it via 496c000d7b91c73b4571637917bc8d64a46392aa (not yet published)

**Comment by dvd101x** [comment]
Thanks Jos!

**Comment by josdejong** [comment]
Fix is published now in `v11.5.1`.
